src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -241,6 +241,13 @@ public function __call( $name, $arguments ) {
 		if ( in_array( $name, $this->compat_methods, true ) ) {
 			return $this->$name( ...$arguments );
 		}
+
+		$class = get_class( $this );
+		$trace = debug_backtrace(); // phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.NeedsInspection
+		$file  = $trace[0]['file'];
+		$line  = $trace[0]['line'];
+		trigger_error( "Call to undefined method $class::$name() in $file on line $line", E_USER_ERROR );
```

Comment:
Can this be extracted to a util/helper class as a static method? Then you could just use that helper in all the other places to reduce code duplication?

Reply:
It could be, but I didn't see it as worth complicating the code by going back up the stack to the calling method.

Reply:
Sure, but what if down the line somebody decides that the wording of the message needs to be changed to make it more clear, or some implementation details should change. Then you'd need to go through the entire codebase to change it instead only in one place.

I'm always for more clarity than duplication (the performance overhead is usually really negligible).